### ISSUE TITLE
problem of train.py 

### ISSUE THREAD

**Issue by kkhuang1990** [isht7/pytorch-deeplab-resnet#42]
In line 223 of train.py file. input and target should be both out[0] and label[0] if we want to use data after multi-scale fusion for training.

**Comment by kkhuang1990** [comment]
Also for esize_label_batch() function in train.py, instead of UpsamplingBilinear2d but UpsamplingNearest2d should be used for resizing labels since labels can only be integer, converting float to int may cause mis-labeling for ground truth labels
